templates/haproxy.cfg.j2

```diff
@@ -71,10 +70,10 @@ frontend openshift-api-server
     bind *:6443
     default_backend openshift-api-server
     mode tcp
-    option tcplog
+    option httplog
 
 backend openshift-api-server
-    balance source
+    balance roundrobin
```

Comment:
Any reason you're using roundrobin instead of source?

Reply:
Just my personal favourite. Ideal for short lived sessions. 'leastconn' could be more applicable in this case.

Reply:
Actually, I like the idea of `leastcon`! :+1: